src/eipmi_response.erl

```diff
@@ -383,10 +383,10 @@ decode_chassis(?GET_CHASSIS_CAPABILITIES, <<?EIPMI_RESERVED:4, L:1, D:1, F:1,
             <<Bridge:8>> ->
                [{bridge_address, Bridge}]
            end,
-    [{interlock, L},
-     {lockout, F},
-     {diagnostic, D},
-     {intrusion, I},
+    [{interlock, case L of 1 -> true; 0 -> false end},
+     {lockout, case F of 1 -> true; 0 -> false end},
+     {diagnostic, case D of 1 -> true; 0 -> false end},
+     {intrusion, case I of 1 -> true; 0 -> false end},
      {fru_address, Fru},
```

Comment:
Would you mind if I change these to use `eipmi_util:get_bool/1`?

Reply:
Go ahead! I didn't realize it was there, but I saw that pattern used elsewhere.